src/mavsdk/plugins/transponder/include/plugins/transponder/transponder.h

```diff
@@ -107,6 +107,7 @@ class Transponder : public PluginBase {
         std::string callsign{}; /**< @brief The callsign */
         AdsbEmitterType emitter_type{}; /**< @brief ADSB emitter type. */
         uint32_t squawk{}; /**< @brief Squawk code. */
+        uint8_t tslc{}; /**< @brief Time Since Last Communication. */
```

Comment:
This file is auto-generated, you have to make that change in the corresponding proto file :+1:

Reply:
I created https://github.com/mavlink/MAVSDK-Proto/pull/280

How does it work? You merge pull request in MAVSDK-proto, re-generate files and commit them in MAVSDK?

Reply:
Yes :+1:. If that was about bigger changes, you would re-generate files here without merging into MAVSDK-proto (you can point the proto submodule to your branch), but in that case it's simple enough we can merge it directly in MAVSDK-proto.